virt/live_migration/virt-initiating-live-migration.adoc

```diff
@@ -0,0 +1,29 @@
+:_content-type: ASSEMBLY
+[id="virt-initiating-live-migration"]
+= Initiating live migration
+include::_attributes/common-attributes.adoc[]
+:context: virt-initiating-live-migration
+
+toc::[]
+
+You can initiate the live migration of a virtual machine (VM) to another node by using the xref:../../virt/live_migration/virt-initiating-live-migration.adoc#virt-initiating-vm-migration-web_virt-initiating-live-migration[{product-title} web console] or the xref:../../virt/live_migration/virt-initiating-live-migration.adoc#virt-initiating-vm-migration-cli_virt-initiating-live-migration[command line].
+
+You can cancel a live migration by using the xref:../../virt/live_migration/virt-initiating-live-migration.adoc#virt-cancelling-vm-migration-web_virt-initiating-live-migration[web console] or the xref:../../virt/live_migration/virt-initiating-live-migration.adoc#virt-cancelling-vm-migration-cli_virt-initiating-live-migration[command line]. The VM remains on its original node.
+
+[TIP]
+====
+You can also initiate and cancel live migration by using the `virtctl migrate <vm_name>` and `virtctl migrate-cancel <vm_name>` commands.
+====
+
+include::modules/virt-initiating-vm-migration-web.adoc[leveloffset=+1]
+
+include::modules/virt-cancelling-vm-migration-web.adoc[leveloffset=+2]
+
+include::modules/virt-initiating-vm-migration-cli.adoc[leveloffset=+1]
+
+include::modules/virt-cancelling-vm-migration-cli.adoc[leveloffset=+2]
```

Comment:
I'm not sure if "canceling" should be a child of "initiating" but I think I see what you're doing (initiating it is a prereq of canceling it, in a way). This doesn't feel like an urgent thing to puzzle out, so I'll leave it at that. :)

Reply:
+1. Thanks for spotting this. I added a couple new headings. I think it makes more sense now. I was starting to get tired....